site/content/docs/4.3/examples/offcanvas/index.html

```diff
@@ -2,20 +2,20 @@
 layout: examples
 title: Offcanvas template
 extra_css:
-  - "offcanvas.css"
-extra_js:
-  - src: "offcanvas.js"
-body_class: "bg-light"
+- "offcanvas.css"
 ---
```

Comment:
This entire file needs to be rethought. There's no reason to remove the custom page content from this. I imagine this should instead be updated to leave the page content as-is, and change the behavior of the navigation sidebar.

Reply:
@mdo 

Sorry, I'm not sure I understood what you propose